WordPress/Classes/Models/BasePost.m

```diff
@@ -142,9 +177,30 @@ - (NSDate *)dateCreated
     return self.date_created_gmt;
 }
 
+// TODO: Double check to make sure this logic matches wp-admin
```

Comment:
For TODOs we should either resolve the comment or create an issue for it and reference it in the TODO comment (that way we know it won't be lost in the code)